tests/dashbio_demos/app_speck.py

```diff
@@ -43,67 +130,118 @@ def layout():
-                         'value': '{}methane.xyz'.format(DATAPATH)}
+                        {'label': 'Default/reset',
+                         'value': 'default'},
+                        {'label': 'Toon',
+                         'value': 'toon'},
                     ],
-                    value='{}dna.xyz'.format(DATAPATH)
+                    value='default'
                 ),
-
+                html.Br(),
+                "2: Select atom style",
                 dcc.Dropdown(
-                    id='speck-preset-dropdown',
-                    options=[
-                        {'label': 'Default', 'value': 'default'},
-                        {'label': 'Ball-and-stick', 'value': 'stickball'},
-                        {'label': 'Toon', 'value': 'toon'},
-                        {'label': 'Licorice', 'value': 'licorice'}
-                    ]
-                ),
-
-                html.Hr(),
-
-                dcc.Checklist(
-                    id='speck-scroll-zoom-enable',
+                    id='speck-preset-atom-style-dropdown',
+                    className='speck-dropdown',
                     options=[
-                        {'label': 'Scroll to zoom', 'value': 'scrollzoom'}
+                        {'label': 'Ball-and-stick',
+                         'value': 'stickball'},
+                        {'label': 'Licorice',
+                         'value': 'licorice'}
                     ],
-                    values=['scrollzoom']
+                    value='default'
                 ),
 
-                html.Div(
-                    id='speck-zoom-slider-container', children=[
-                        "Zoom molecule",
-                        html.Br(),
-                        dcc.Slider(
-                            id='speck-zoom-slider',
-                            min=0,
-                            max=0.1,
-                            step=0.0001,
-                            value=0.02,
-                        ),
-                    ]
+                dcc.Store(
+                    id='speck-store-preset-rendering',
+                    data=None
+                ),
+                dcc.Store(
+                    id='speck-store-preset-atom-style',
+                    data=None
+                ),
+                dcc.Store(
+                    id='speck-view-updated',
+                    data=None
                 ),
 
-                html.Hr(),
+                html.Div(
+                    id='speck-idk'
+                )
             ]
         ),
 
+        html.Div(
+            id='speck-controls-detailed',
+            className='speck-controls',
+            children=default_sliders
+        )
 
     ])
 
 
 def callbacks(app):  # pylint: disable=redefined-outer-name
 
+    @app.callback(
+        Output('speck-controls-detailed', 'style'),
+        [Input('speck-enable-presets', 'values')]
+    )
+    def show_hide_detailed_controls(presets_enable):
+        if len(presets_enable) > 0:
```

Comment:
I would be more in favor to use [one return statement per function](https://docs.python-guide.org/writing/style/#returning-values)